src/utils/react-18-compat.ts

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+
+/**
+ * This interface needs to be added to a property type that is used on `React.Component`.
+ *
+ * With React 18, React.Component doesn't include children types anymore.
+ */
+export interface ReactComponentDefaultProps {
+  children?: React.ReactNode | undefined;
```

Comment:
isn't this redundant? `children?: React.ReactNode;` should be enough

Reply:
This type is everywhere in the react official types and I wanted to stick with it:

<img width="630" alt="screenshot-15-11-2022-at-15-47-07@2x" src="https://user-images.githubusercontent.com/7814926/201948977-a7aed407-233b-4604-932f-d262edf197fa.png">